app/routes.py

```diff
@@ -1,17 +1,20 @@
 from flask import render_template, url_for, flash, redirect, request
 from flask_login import current_user, login_user, logout_user, login_required
 from werkzeug.urls import url_parse
+from werkzeug.datastructures import CombinedMultiDict
 
-from app.forms import RegistrationForm, UploadJokeForm, LoginForm
-from app.models import User, Joke
+
+from app.forms import RegistrationForm, UploadJokeForm, LoginForm, EpisodeUploadForm
+from app.models import User, Joke, Episode
 from app import app, db
 from app.utils import admin_required
 
 
 @app.route('/')
 def index():
     feed_blank = 'Podcast Main page: RSS feed'
-    return render_template('index.html', feed_blank=feed_blank)
+    form = EpisodeUploadForm()
```

Comment:
You need to check user is login

Reply:
user login check in upload_podcast_handle

Reply:
i create upload route for registered user